src/test/java/core/basesyntax/StorageTest.java

```diff
@@ -0,0 +1,158 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class StorageTest {
+    private static final String testLogin = "Kolya";
+    private static final String testPassword = "password";
+    private static final Integer testAge = 23;
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User user = new User();
```

Comment:
Lets remove this global user, and create new one every time in test method

Reply:
OK, but let's talk about it at QnA)